numba_cuda/numba/cuda/cuda_paths.py

```diff
@@ -196,6 +196,9 @@ def _get_nvrtc_wheel():
                     raise RuntimeError(
                         f'Path does not exist: "{builtins_path}"'
                     )
+                # Add the directory containing the builtins DLL to the DLL search path
+                # We already have a side effect of loading the nvrtc DLL by the time we get here and this function is only called lazily if nvrtc isn't found elsewhere
+                os.add_dll_directory(os.path.dirname(builtins_path))
```

Comment:
This is a quick and dirty temporary workaround until we move to using `cuda-pathfinder` to handle finding and loading nvrtc more generally. Alternatively, I could load this `DLL` via a `CDLL` call similar to what is done above for the nvrtc DLL (in order to get the handle), but I figured we'd be better off just setting the directory to let nvrtc load it itself.